### DISCUSSION TITLE
Accuracy not improving on CIFAR-10 dataset

### DISCUSSION THREAD

**Discussion by sauravtii** [jeshraghian/snntorch#129] [General]
Hello!

I am trying out tutorial_6 (https://colab.research.google.com/github/jeshraghian/snntorch/blob/master/examples/tutorial_6_CNN.ipynb) but the only difference is I am using CIFAR-10 dataset instead of MNIST. I trained it for 10 epochs but the accuracy wasn't good, so I tried training for 50 epochs and the accuracy is still around 35-40 %. Can anyone please guide me on resolving this issue ?

**Comment by ahenkes1** [comment]
Have you tried a larger network?

**Comment by jeshraghian** [comment]
Converted to a discussion. 

I've been able to train a dense SNN with one hidden layer to hit around 50%, so a CSNN should exceed that. Consider a larger model, more time steps, adding batchnorm, using a learning rate schedule, etc.
